src/cache.ts

```diff
@@ -15,19 +14,39 @@ const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 const log = logging.createLogger('cache');
 
 /**
- * Implements access to CMake cache entries. See `api.CacheEntry` for more
- * information.
+ * The type of a CMake cache entry
  */
-export class Entry implements api.CacheEntry {
-    public readonly type: api.CacheEntryType = api.CacheEntryType.Uninitialized;
+export enum CacheEntryType {
+    Bool = 0,
+    String = 1,
+    Path = 2,
+    FilePath = 3,
+    Internal = 4,
+    Uninitialized = 5,
+    Static = 6,
+}
+
+/**
+ * Implements access to CMake cache entries.
+ */
+export class Entry {
```

Comment:
Can we call this class `CacheEntry` instead?  It would solidify meaning in a number of places where we import the class name directly and just call it `Entry` instead of `cache.Entry`.  VS Code "rename" feature should make this easy on you.

Reply:
Renamed.